src/OrbitVulkanLayer/SubmissionTracker.h

```diff
@@ -1029,7 +980,16 @@ class SubmissionTracker : public VulkanLayerProducer::CaptureStatusListener {
   absl::flat_hash_map<VkCommandBuffer, VkDevice> command_buffer_to_device_;
 
   absl::flat_hash_map<VkCommandBuffer, CommandBufferState> command_buffer_to_state_;
-  absl::flat_hash_map<VkQueue, std::vector<QueueSubmission>> queue_to_submissions_;
+
+  static constexpr auto kPreSubmissionCPUTimestampComparator =
```

Comment:
Prefer "Cpu" to "CPU" as per styleguide's "prefer to capitalize abbreviations as single words, e.g., StartRpc() rather than StartRPC()".

Reply:
Done